NullPointerException when setting "addedToClassPath=true" without specifying output directory

`Caused by: java.lang.NullPointerException
	at org.codehaus.mojo.xml.TransformMojo.addToClasspath(TransformMojo.java:253)
	at org.codehaus.mojo.xml.TransformMojo.transform(TransformMojo.java:569)
	at org.codehaus.mojo.xml.TransformMojo.execute(TransformMojo.java:632)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more`

--- a/src/main/java/org/codehaus/mojo/xml/TransformMojo.java
+++ b/src/main/java/org/codehaus/mojo/xml/TransformMojo.java
@@ -566,7 +566,7 @@ public class TransformMojo
 
         if ( pTransformationSet.isAddedToClasspath() )
         {
-            addToClasspath( pTransformationSet.getOutputDir() );
+            addToClasspath( getOutputDir( pTransformationSet.getOutputDir() ));
         }
     }
 

--- a/src/test/xinclude-xsl/pom.xml
+++ b/src/test/xinclude-xsl/pom.xml
@@ -32,6 +32,7 @@
 			<configuration>
 				<transformationSets>
 					<transformationSet>
+                                                <addedToClasspath>true</addedToClasspath>
 						<xincludeAware>true</xincludeAware>
 						<dir>xml</dir>
 						<stylesheet>xsl/copy.xsl</stylesheet>
@@ -41,6 +42,7 @@
 					</transformationSet>
 
 					<transformationSet>
+                                                <addedToClasspath>true</addedToClasspath>
 						<xincludeAware>false</xincludeAware>
 						<dir>xml</dir>
 						<stylesheet>xsl/copy.xsl</stylesheet>

